es55x/src/main/scala/tech/beshu/ror/es/IndexLevelActionFilter.scala

```diff
@@ -150,8 +159,8 @@ class IndexLevelActionFilter(settings: Settings,
   }
 
   private def startRorInstance() = doPrivileged {
-    new Ror(RorMode.Plugin)
-      .start(env.configFile, new EsAuditSinkService(client), new EsIndexJsonContentService(client))
+    new Ror(RorMode.Plugin, auditSinkCreator)
```

Comment:
don't you think we have an inconsistency here? We pass IndexJsonContentService implementation in the start method but AuditSinkCreator in the constructor. Can we pick one approach?

Reply:
Your suggestion encouraged me to do some changes:

* convert `ReadonlyRest` trait to class, no more `Ror` class
* remove all params from `start` method, pass everything in constructor of `ReadonlyRest`
* extract `RorInstance` to separate file
* extract ssl tests from `ReadonlyRestStartingTests` to `SslConfigurationTest`

WDYT?

Reply:
it's much better now.